.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+-Xmx2048m
```

Comment:
This is great, but since it's just a max, should it err higher, even? 4096 or more?